protocols/identify/CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
 
 - Update to `libp2p-swarm` `v0.40.0`.
 
+- Change default `cache_size` of `Config` to 100. See [PR 2995].
```

Comment:
With `v0.49.0` released, this would need to bump the `libp2p-identify` version, add a `v0.41.0` changelog section, add an entry and adjust the root level `Cargo.toml` and `CHANGELOG.md` accordingly. 

See https://github.com/libp2p/rust-libp2p/blob/master/docs/release.md

I am sorry for the trouble @efarg.

Reply:
lmao no problem. I'll update.